x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.tsx

```diff
@@ -40,6 +40,7 @@ export interface BodyProps {
   columnRenderers: ColumnRenderer[];
   data: TimelineItem[];
   docValueFields: DocValueFields[];
+  expanded: ActiveTimelineExpandedEvent;
```

Comment:
I do not really like how we define the type of `ActiveTimelineExpandedEvent` to me, you cannot have eventId without indexName. Tell me if you see it the same way, I will remove the conditional attributes in `ActiveTimelineExpandedEvent` and make `expanded` as a conditional attribute.